docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -1,184 +1,136 @@
-
 ```shell
-$ kubectl create -f docs/user-guide/horizontal-pod-autoscaling/hpa-php-apache.yaml
-horizontalpodautoscaler "php-apache" created
-```
-
-Alternatively, we can create the autoscaler using [kubectl autoscale](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/user-guide/kubectl/kubectl_autoscale.md).
-The following command will create the equivalent autoscaler as defined in the [hpa-php-apache.yaml](/docs/user-guide/horizontal-pod-autoscaling/hpa-php-apache.yaml) file:
-
-```shell
-$ kubectl autoscale rc php-apache --cpu-percent=50 --min=1 --max=10
-replicationcontroller "php-apache" autoscaled
+$ kubectl autoscale deployment php-apache --cpu-percent=50 --min=1 --max=10
+deployment "php-apache" autoscaled
 ```
 
 We may check the current status of autoscaler by running:
 
 ```shell
 $ kubectl get hpa
-NAME         REFERENCE                                   TARGET    CURRENT   MINPODS   MAXPODS   AGE
-php-apache   ReplicationController/default/php-apache/   50%       0%        1         10        27s
+NAME         REFERENCE                     TARGET    CURRENT   MINPODS   MAXPODS   AGE
+php-apache   Deployment/php-apache/scale   50%       0%        1         10        18s
+
 ```
 
 Please note that the current CPU consumption is 0% as we are not sending any requests to the server
-(the ``CURRENT`` column shows the average across all the pods controlled by the corresponding replication controller).
+(the ``CURRENT`` column shows the average across all the pods controlled by the corresponding deployment).
 
 ## Step Three: Increase load
 
 Now, we will see how the autoscaler reacts on the increased load of the server.
-We will start an infinite loop of queries to our server (please run it in a different terminal):
+We will start a container with busybox image and an infinite loop of queries to our server inside (please run it in a different terminal):
 
 ```shell
-$ while true; do curl http://146.148.6.244; done
+$ kubectl run -i --tty load-generator --image=busybox /bin/sh
+
+Hit enter for command prompt
+
+$ while true; do wget -q -O- http://php-apache.default.svc.cluster.local; done
 ```
 
-We may examine, how CPU load was increased (the results should be visible after about 3-4 minutes) by executing:
+We may examine, how CPU load was increased by executing (it usually takes 1 minute):
 
 ```shell
 $ kubectl get hpa
-NAME         REFERENCE                                   TARGET    CURRENT   MINPODS   MAXPODS   AGE
-php-apache   ReplicationController/default/php-apache/   50%       305%      1         10        4m
+NAME         REFERENCE                     TARGET    CURRENT   MINPODS   MAXPODS   AGE
+php-apache   Deployment/php-apache/scale   50%       305%      1         10        3m
+
 ```
 
 In the case presented here, it bumped CPU consumption to 305% of the request.
-As a result, the replication controller was resized to 7 replicas:
+As a result, the deployment was resized to 7 replicas:
```

Comment:
Let's add a comment that it may take few steps with some stabilization in between to get to stable number of replicas.